### ISSUE TITLE
CAP_CAP_SYS_CHROOT causes svc strategy to fail

### ISSUE THREAD

**Issue by aarononeal** [utkuozdemir/pv-migrate#165]
@Frankkkkk, @utkuozdemir
Commit 610e82c is failing with:

```
Error: failed to create containerd task: OCI runtime create failed: container_linux.go:367: starting container process caused: unknown capability "CAP_CAP_SYS_CHROOT": unknown
```

Because apparently the `CAP_` prefix is added automatically and is resulting in a duplicate. It appears the value should just be `SYS_CHROOT`.

**Comment by Frankkkkk** [comment]
Indeed; reading [the documentation](https://kubernetes.io/docs/concepts/security/pod-security-policy/#capabilities) it seems that the `CAP_` prefix should not be added.

It looks like some "intelligent" CRIs like containerd supporting having the extra prefix whereas others dont

**Comment by utkuozdemir** [comment]
Many thanks for the report and the coordination guys. Good catch!

Let me fix this quickly and make a release :)
